Samples/iOS-SwiftUI/iOS-SwiftUI-UITests/LaunchUITests.swift

```diff
@@ -48,6 +49,15 @@ class LaunchUITests: XCTestCase {
 
         XCTAssertEqual(app.staticTexts["TTDInfo"].label, "TTID and TTFD found")
     }
+  
+  func testCaptureErrorReturnsValidId() {
+    let app = newAppSession()
+    app.safelyLaunch()
+    app.buttons["Capture Error"].tap()
+    let errorId = app.staticTexts["errorId"].label
+    let sentryErrorId = SentryId(uuidString: errorId)
+    XCTAssertNotEqual(sentryErrorId.sentryIdString, SentryId.empty.sentryIdString)
```

Comment:
`l`: I would add an assertion message to explain that we expect the ID to be anything but an empty ID, because the SDK returns an empty ID if the capture method didn't work.